conf/modules_illumina.config

```diff
@@ -0,0 +1,1073 @@
+========================================================================================
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+def variant_caller = params.variant_caller
+if (!variant_caller) { variant_caller = params.protocol == 'amplicon' ? 'ivar' : 'bcftools' }
+
+def assemblers = params.assemblers ? params.assemblers.split(',').collect{ it.trim().toLowerCase() } : []
+
+//
+// Pre-processing and general configuration options
+//
+
+process {
+    withName: '.*:.*:PREPARE_GENOME:GUNZIP_.*' {
+        publishDir = [
+            path: { "${params.outdir}/genome" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.save_reference
+        ]
+    }
+
+    withName: '.*:.*:PREPARE_GENOME:UNTAR_.*' {
+        ext.args2 = '--no-same-owner'
+        publishDir = [
+            path: { "${params.outdir}/genome" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.save_reference
+        ]
+    }
+
+    withName: 'CAT_FASTQ' {
+        publishDir = [
+            path: { "${params.outdir}/fastq" },
+            enabled: false
+        ]
+    }
+}
+
+if (!params.skip_fastqc) {
+    process {
+        withName: '.*:.*:FASTQC_FASTP:FASTQC_RAW' {
+            ext.args = '--quiet'
+            publishDir = [
+                path: { "${params.outdir}/fastqc/raw" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+        }
+    }
+}
+
+if (!params.skip_fastp) {
+    process {
+        withName: 'FASTP' {
+            ext.args = '--cut_front --cut_tail --trim_poly_x --cut_mean_quality 30 --qualified_quality_phred 30 --unqualified_percent_limit 10 --length_required 50'
+            publishDir = [
+                [
+                    path: { "${params.outdir}/fastp" },
+                    mode: 'copy',
+                    pattern: "*.{json,html}"
+                ],
+                [
+                    path: { "${params.outdir}/fastp/log" },
+                    mode: 'copy',
+                    pattern: "*.log"
+                ],
+                [
+                    path: { "${params.outdir}/fastp" },
+                    mode: 'copy',
+                    pattern: "*.fail.fastq.gz",
+                    enabled: params.save_trimmed_fail
+                ]
+            ]
+        }
+
+        withName: 'MULTIQC_TSV_FAIL_READS' {
```

Comment:
I guess here is not a problem since the `MULTIQC` not aliased process does indeed set the same path, but maybe it would be better to alias it to `MULTIQC_MAIN` to avoid any issue?